### PR TITLE
Add documentation for new FieldDescriptors (RN35278)

### PR DESCRIPTION
**Pull request by Seppe-Vanhee**
Added description of new FieldDescriptors that will be added in RN35278

### CODE REVIEW

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 
 - **DomInstanceValueFieldDescriptor**: Available from DataMiner 10.2.3/10.3.0 onwards. Can be used to define that a field should contain the ID of a `DomInstance`. However, compared to the `DomInstanceFieldDescriptor`, this one also references a specific value of that `DomInstance`. The configuration is the same as the other descriptor, but it adds the *FieldDescriptorId* property that references a specific `FieldValue`.
 
+- **GroupFieldDescriptor**: Available from DataMiner 10.3.2 onwards. Can be used to define that a field should contain the name of a DataMiner user group.
```
Comment (Skyline-ThomasGH) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:71]:
```suggestion
- **GroupFieldDescriptor**: Available from DataMiner 10.3.2/10.4.0 onwards. Can be used to define that a field should contain the name of a DataMiner user group.
```

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 2]
```diff
@@ -68,6 +68,10 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
...
+
+- **UserFieldDescriptor**: Available from DataMiner 10.3.2 onwards. Can be used to define that a field should contain the name of a DataMiner user. There is a *GroupId* property that can be used to define which group the user should be a part of.
```
Comment (Skyline-ThomasGH) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:73]:
```suggestion
- **UserFieldDescriptor**: Available from DataMiner 10.3.2/10.4.0 onwards. Can be used to define that a field should contain the name of a DataMiner user. There is a *GroupId* property that can be used to define which group the user should be a part of. 
```

**Review by Skyline-ThomasGH — APPROVED** [review]
Also add the main release versions just like the others.

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 3]
```diff
@@ -78,6 +82,7 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 > - ServiceDefinitionFieldDescriptor
 >
 > From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*.
+> From DataMiner 10.3.2 onwards, this also applies for the *UserFieldDescriptor*
```
Comment (VanRobaysWouter) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:85]:
```suggestion
> From DataMiner 10.3.2/10.4.0 onwards, this also applies for the *UserFieldDescriptor*
```

**Comment by MariekeGO** [comment]
As this release note currently hasn't been merged to any DataMiner version yet, I've reverted this PR to a draft for now. Could you mark it as "Ready for review" as soon as the RN has been merged?

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 4]
```diff
@@ -68,6 +68,10 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 
 - **DomInstanceValueFieldDescriptor**: Available from DataMiner 10.2.3/10.3.0 onwards. Can be used to define that a field should contain the ID of a `DomInstance`. However, compared to the `DomInstanceFieldDescriptor`, this one also references a specific value of that `DomInstance`. The configuration is the same as the other descriptor, but it adds the *FieldDescriptorId* property that references a specific `FieldValue`.
 
+- **GroupFieldDescriptor**: Available from DataMiner 10.3.2/10.4.0 onwards. Can be used to define that a field should contain the name of a DataMiner user group.
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:71]: 10.3.2 should be updated to 10.3.3

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 5]
```diff
@@ -68,6 +68,10 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
...
+
+- **UserFieldDescriptor**: Available from DataMiner 10.3.2/10.4.0 onwards. Can be used to define that a field should contain the name of a DataMiner user. There is a *GroupId* property that can be used to define which group the user should be a part of.
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:73]: 10.3.2 should be updated to 10.3.3

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 6]
```diff
@@ -77,7 +81,7 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 > - ReservationFieldDescriptor
 > - ServiceDefinitionFieldDescriptor
 >
-> From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*.
+> From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*, and from DataMiner 10.3.2/10.4.0 onwards, this also applies for the *UserFieldDescriptor*.
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:84]: This change can be reverted again

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 7]
```diff
@@ -77,7 +81,7 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 > - ReservationFieldDescriptor
 > - ServiceDefinitionFieldDescriptor
 >
-> From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*.
+> From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*, and from DataMiner 10.3.3/10.4.0 onwards, this also applies for the *UserFieldDescriptor*.
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:84]:
```suggestion
> From DataMiner 10.2.5/10.3.0 onwards, this also applies for the *DomInstanceValueFieldDescriptor*.
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**